aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc4/intro-to-aspnet-mvc-4.md

```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-aspnet-mvc4/intro-to-aspnet-mvc-4
 title: "Intro to ASP.NET MVC 4 | Microsoft Docs"
 author: Rick-Anderson
-description: "An updated version if this tutorial is available here using Visual Studio 2013. The new tutorial uses ASP.NET MVC 5, which provides many improvements over t..."
+description: "An updated version if this tutorial is available here using Visual Studio 2013. The new tutorial uses ASP.NET MVC 5, which provides users many improvements over t..."
```

Comment:
Let's fix the grammar error on the first sentence and cut off the second sentence so it's not truncated.  Like this, "An updated version **of** this tutorial is available here using Visual Studio 2013. The new tutorial uses ASP.NET MVC 5."